gamemodes/terrortown/entities/weapons/weapon_ttt2_lens.lua

```diff
@@ -419,7 +419,11 @@ else
 		tbl.curtime = CurTime()
 		tbl.angle = ang.y
 		tbl.normal = tr.HitNormal
-		tbl.col = ply:GetRoleColor()
+		if ply:GetSubRole() ~= ROLE_NONE then
+			tbl.col = ply:GetRoleColor()
+		else
+			tbl.col = Color(80, 173, 59, 255)
```

Comment:
This color is also available via the role table/ module, isnt it @TimGoll ?

Reply:
My bad, fixed that

Reply:
You might want to switch to the new role module, but I'm currently unsure. Just to be safe, I will ping @Alf21 to get his opinion.

Reply:
Went ahead and switched it to the new roles module.